app/controllers/users_controller.rb

```diff
@@ -27,7 +27,7 @@ def create
 
   def edit
```

Comment:
The advantage of modularizing your authorization logic is that you can apply it more evenly and consistently--not much good to protect the edit action and not the update, for instance.